test/test_optimize.py

```diff
@@ -28,6 +28,7 @@ def mode(request):
     ('ipopt', ''),
     ('dlib', 'default'),
     ('pyswarm', ''),
+    ('cma', ''),
```

Comment:
maybe name it cmaes

Reply:
In this file only?

Reply:
should be only this file. just use "cmaes" / "Cmaes" everywhere.